packages/tradershub/src/components/TotalAssets/TotalAssets.tsx

```diff
@@ -3,19 +3,19 @@ import { twMerge } from 'tailwind-merge';
 import { TotalAssetsLoader } from '@/components';
 import { useTotalAssets } from '@/hooks';
 import { useActiveTradingAccount } from '@deriv/api';
-import { Text, useDevice } from '@deriv-com/ui';
+import { Text } from '@deriv-com/ui';
 
 const TotalAssets = () => {
     const { data: totalAssets, isSuccess } = useTotalAssets();
     const { data: activeTrading } = useActiveTradingAccount();
 
-    const { isDesktop } = useDevice();
-
     if (!isSuccess) return <TotalAssetsLoader />;
 
     return (
         <div className='relative inline-block w-auto text-center lg:text-right'>
-            {isDesktop && <Text size='sm'>Total assets</Text>}
+            <div className='d-none lg:block'>
+                <Text size='sm'>Total assets</Text>
+            </div>
```

Comment:
👍🏼 :D
but actually if you don't wrap it in a div it should work also right?

Reply:
Should work, but since this condition is in tradershub side its better to add a div rather than telling Text component to do the condition